boltdesignsystem/bolt
# DS-174 Layout

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-174](https://pegadigitalit.atlassian.net/browse/DS-174)

## Summary

Create Layout in the layouts directory.

## Details

1. Layout is used in a Drupal Admin UI POC which removes all wrappers, bands, grids, and nested paragraphs.
2. It uses CSS grid to handle all common design layouts.
3. It significantly reduces the layers of HTML in general page layouts.

## How to test

Run the branch locally and read all the Layout docs. Make sure all props work as expected.
